index.html

```diff
@@ -17,15 +17,25 @@
     <div class="Main-container">
         <div class="container">
             <header class="header hoverable">
```

Comment:
- [ ] please it would be better if you remove that line in the top 
![image](https://user-images.githubusercontent.com/61790587/167230121-0a0ff466-6abe-4563-b4d7-f894cdd733b0.png)